modules/labs-react/radio/lib/RadioLabel.tsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+
+import {Themeable, createSubcomponent, ExtractProps} from '@workday/canvas-kit-react/common';
+import {useRadioModel} from './hooks/useRadioModel';
+import {Flex} from '@workday/canvas-kit-react/layout';
+import {RadioInput} from './RadioInput';
+import {RadioText} from './RadioText';
+
+interface RadioLabelContextInterface {
+  disabled?: boolean | undefined;
+  variant?: 'inverse' | undefined;
+  checked?: boolean;
+  value?: string | number;
+}
+export interface RadioLabelProps
+  extends Themeable,
+    ExtractProps<typeof Flex, never>,
+    RadioLabelContextInterface {
```

Comment:
Looking for feedback here for the props/ context. `disabled` is used to disable the input and the style the color text. `variant` exist at the top level so that makes sense. I don't know if `value` and `checked` makes sense. But I kept it there because when we build the `Radio` component, I can still pass those properties down. When using the compound component version, you wouldn't add `checked` and value` on `RadioGroup.Label`. But when I use `RadioGroup.Radio` i still need to be able to pass those props down.